src/compiler/scala/tools/nsc/backend/opt/ConstantOptimization.scala

```diff
@@ -28,8 +28,8 @@ abstract class ConstantOptimization extends SubComponent {
   import global._
   import icodes._
   import icodes.opcodes._
-  import scala.tools.nsc.backend.icode.Primitives._
-  import scala.tools.nsc.backend.icode.Opcodes._
+  import scala.tools.nsc.backend.jvm.Primitives._
```

Comment:
`icode.Primitives` contained both a class and an object, I cannot compile the class with dotty and I need to compile the object, so I had to move the object. I decided to move it to `jvm.Primitives` instead of something like `icode.PrimitivesOps` because that way I don't need to compile anything from the `icode` subdirectory which makes the selection with filters of files compiled by dotty simpler.